lib/Doctrine/ruleset.xml

```diff
@@ -17,6 +19,7 @@
         <!-- But allow empty catch -->
         <exclude name="Generic.CodeAnalysis.EmptyStatement.DetectedCatch"/>
     </rule>
+    <rule ref="Generic.CodeAnalysis.ForLoopShouldBeWhileLoop"/>
```

Comment:
What is this rule?

Reply:
oh sry, that was something else I accidently commited xD. will fix it